### PR TITLE
Fix for #277 unhandled 'pagure.issue.drop' messages

### PR DESCRIPTION
**Pull request by Ghost-script**
@ralphbean , please review this when you get time. :smiley:

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/pagure.py [hunk 1]
```diff
@@ -1623,6 +1623,70 @@ class TestGitCommit(Base):
          'topic': 'io.pagure.prod.pagure.git.receive'
     }
 
+class TestIssueDrop(Base):
+    """ These messages are published when a ticket is closed against a
```
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/pagure.py]: Is it when they are closed?  Or when they are deleted? or..?  Can we explain more what "drop" means here?

Comment (Ghost-script) [inline fedmsg_meta_fedora_infrastructure/tests/pagure.py]: I think it means `delete` .
https://github.com/pypingou/pagure/blob/e1497f5e98387fa547e251ccf02ba179adc95202/pagure/ui/issues.py#L532

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/pagure.py]: OK, can we changed the subtitle to read ``yangl1996 deleted ticket docs-test#42: "foobar"'``?  :+1: pending that.

**Comment by Ghost-script** [comment]
@ralphbean , fixed that. Good to be merged?

**Comment by ralphbean** [comment]
Looks great, thank you!  :)

**Pull request merged**